powerlevel9k.zsh-theme

```diff
@@ -531,12 +531,14 @@ prompt_node_version() {
 # Node version from NVM
 # Only prints the segment if different than the default value
 prompt_nvm() {
-  local node_version=$(nvm current)
-  local nvm_default=$(cat $NVM_DIR/alias/default)
-  [[ -z "${node_version}" ]] && return
-  [[ "$node_version" =~ "$nvm_default" ]] && return
+  if [[ $(type nvm) =~ 'nvm is a shell function'* ]]; then
+    local node_version=$(nvm current)
+    local nvm_default=$(cat $NVM_DIR/alias/default)
+    [[ -z "${node_version}" ]] && return
+    [[ "$node_version" =~ "$nvm_default" ]] && return
+  fi
 
-  $1_prompt_segment "$0" "$2" "green" "011" "${node_version:1}" 'NODE_ICON'
+  [[ -n "${node_version}" ]] && $1_prompt_segment "$0" "$2" "green" "011" "${node_version:1}" 'NODE_ICON'
```

Comment:
Would that work? `node_version` is defined in the if statement as a local variable..